ztp/extra-manifests-builder/04-accelerated-container-startup/accelerated-container-startup.sh

```diff
@@ -80,6 +145,7 @@ resetAffinity() {
     logger "Recovery: Failed to re-affine $failcount processes"
     return 1
   fi
+  resetOVSthreads "$(getCPUCount ${cpuset})"
```

Comment:
I think we probably want to run this code even if `$failcount` implies some taskset operations above failed.  and if 'resetOVSthreads' fails, we should probably increment `((failcount++))`

Reply:
I agree. done